lib/interact-for-custom-metadata.ts

```diff
@@ -0,0 +1,83 @@
+import prompts from "prompts"
+import { getSeam } from "./get-seam"
+import { CustomMetadata } from "@seamapi/types/connect"
+export const interactForCustomMetadata = async (
+  custom_metadata: CustomMetadata
+) => {
+  const seam = await getSeam()
+  const updated_custom_metadata = { ...custom_metadata }
+
+  const displayCurrentCustomMetadata = () => {
+    console.log("Current custom metadata.")
+    if (Object.keys(updated_custom_metadata).length > 0) {
+      Object.keys(updated_custom_metadata).forEach((key, index) => {
+        console.log(`${index + 1}: ${key}: ${updated_custom_metadata[key]}`)
+      })
+    } else {
+      console.log("The custom metadata param is empty.")
+    }
+  }
+
+  let action: string
+
+  do {
+    displayCurrentCustomMetadata()
+
+    const response = await prompts({
+      type: "select",
+      name: "action",
+      message: "Choose an action:",
+      choices: [
+        { title: "Add an item to params", value: "add" },
+        { title: "Remove an item from params", value: "remove" },
+        { title: "Finish editing params", value: "done" },
```

Comment:
```suggestion
        { title: "Remove an item from custom_metadata", value: "remove" },
        { title: "Finish editing custom_metadata", value: "done" },
```

Reply:
I have not committed this changes from params to custom_metadata because it is ambiguous IMO. You are changing the param your are sending, not the custom_metadata itself. So if you "remove" an item here means you are just configuring your param, not removing from the DB.

I'm going to merge this, if you still think we should come back I can make a new PR